dozer/cogs/levels.py

```diff
@@ -265,7 +281,7 @@ async def checkrolelevels(self, ctx):
         e = discord.Embed(title=f"Level roles for {ctx.guild}", color=blurple)
         if roles:
             e.description = f"This server has {len(roles)} level roles"
-            for level_role in roles:
+            for level_role in roles.__reversed__():
```

Comment:
Does this need to be reversed?

Reply:
In order to match the format of the mee6 settings it does, and personally I think it looks better when it does

Reply:
In that case the `reverse=True` argument to `sorted(...)` will accomplish the same thing in a simpler way. But that's just a nitpick.